client/src/views/Simulation/components/ProvenanceGraph/ProvenanceGraph.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="provenance-graph-container" v-if="isOpen">
+    <aside slot="left">
+      <div class="btn-group btn-group-sm">
+        <button
+          class="btn btn-secondary"
+          @click="onClickCondensed"
+          > Condensed
+        </button>
+        <button
+          class="btn btn-secondary"
+          @click="onClickExpanded"
+          > Expanded
+        </button>
+      </div>
+    </aside>
+    <close-button @close="onClose"/>
+    <div class="flex-grow-1 position-relative overflow-scroll hide-scrollbar panel-body">
+      <slot name="content"/>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+  import Component from 'vue-class-component';
+  import Vue from 'vue';
+  import { Prop } from 'vue-property-decorator';
+  import CloseButton from '@/components/widgets/CloseButton.vue';
+
+  const components = {
+    CloseButton,
+  };
+
+  @Component({ components })
+  export default class ProvenanceGraph extends Vue {
+    @Prop({ default: false })
+    isOpen: boolean;
+
+    @Prop({ required: false })
+    activeTabId: string;
+
+    onClose (): void {
+      this.$emit('close-pane');
+    }
+
+    onClickCondensed (): void {
+      this.$emit('toggle-click', 'condensed');
+    }
+
+    onClickExpanded (): void {
+      this.$emit('toggle-click', 'expanded');
+    }
+  }
+</script>
+
+<style scoped>
+.provenance-graph-container {
+  background-color: var(--bg-graphs);
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
```

Comment:
I don't think we need the box shadow, the borders are good enough.
But I let that call be made by @RosaRomeroGomez

Reply:
Thanks, I have removed it